docs/finality-provider-operation.md

```diff
@@ -630,6 +630,8 @@ fpd create-finality-provider \
   --chain-id <chain-id> \
   --eots-pk <eots-pk-hex> \
   --commission-rate 0.05 \
+  --commission-max-rate 0.05 \
+  --commission-max-change-rate 0.05 \
```

Comment:
Why not use the default values here? https://github.com/cosmos/cosmos-sdk/blob/3ec128b4d1518446daed9ae928969a8ba2ebf544/x/staking/client/cli/tx.go#L31